source/code/core/tasks/private/task_thread_v2.cxx

```diff
@@ -0,0 +1,133 @@
+#include <ice/task_thread.hxx>
+#include <ice/task_list.hxx>
+#include <ice/task.hxx>
+#include <ice/os/windows.hxx>
+#include <ice/memory/stack_allocator.hxx>
+#include <thread>
+
+#include "internal_task.hxx"
+
+namespace ice
+{
+
+    class SimpleTaskThread_v2 : public ice::TaskThread_v2
```

Comment:
A tech-debt cleanup is required for the whole coroutine scheduling API. Mostly because with the current approach using Tracy with Fibers is almost impossible. (#95)

Reply:
Can be taken under the umbrella of #112